LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Neontribe
+Copyright (c) 2017 Neontribe
```

Comment:
Should we add this file to the build script as something to retain, as the dates for individual projects may be different?